src/lib/mongoBackend/MongoCommonSubscription.cpp

```diff
@@ -202,33 +202,43 @@ void setEntities(const Subscription& sub, BSONObjBuilder* b)
   BSONArrayBuilder entities;
   for (unsigned int ix = 0; ix < sub.subject.entities.size(); ++ix)
   {
-    EntID en = sub.subject.entities[ix];
+    EntID       en        = sub.subject.entities[ix];
+    std::string finalId;
+    std::string finalType;
+    std::string isIdPattern;
+    bool        isTypePattern = false;
```

Comment:
Yeah, I wasn't considering to change the format of isPattern in mongo or anything like that, just to have the local vars of the same type.

All is better with the comment

NFTM (nothing further to modify :-))